README.md

```diff
@@ -5,6 +5,18 @@ This repo contains most notably:
  - a client CLI tool to fetch and verify drand beacons from the various available sources in your terminal
  - a gossipsub relay to relay drand beacons on gossipsub
 
+# Migration from drand/drand
+
+Prior to drand V2 release, the drand client code lived in the drand/drand repo. Since its V2 release, the drand daemon code aims at being more minimalistic and having as little dependencies as possible.
```

Comment:
```suggestion
Prior to drand V2 release, the drand client code lived in the drand/drand repo. Since its V2 release, the drand daemon code aims at being more minimalist and having as few dependencies as possible.
```